packages/blocks/src/api/utils.js

```diff
@@ -30,6 +30,30 @@ extend( [ namesPlugin, a11yPlugin ] );
  */
 const ICON_COLORS = [ '#191e23', '#f8f9f9' ];
 
+/**
+ * Determines whether the block's attribute is equal to the default attribute
+ * which means the attribute is unmodified.
+ * @param {Object} attributeDefinition The attribute's definition of the block type.
+ * @param {*}      value               The attribute's value.
+ * @return {boolean} Whether the attribute is unmodified.
+ */
+function isAttributeUnmodified( attributeDefinition, value ) {
```

Comment:
Nit: Maybe we should follow naming conventions of similar methods - `isUnmodifiedAttribute`.